Rendera/include/cube.hpp

```diff
@@ -0,0 +1,209 @@
+#ifndef CUBE_HPP
+#define CUBE_HPP
+
+#include "vec3.hpp"
+#include "ray.hpp"
+#include "material.hpp"
+
+#define INFINITY 1E10
```

Comment:
You can use the [`std::numeric_limits`](https://en.cppreference.com/w/cpp/types/numeric_limits) class template in the [`<limits>`](https://en.cppreference.com/w/cpp/header/limits) header instead of this macro.